cilium/ipcache.cc

```diff
@@ -92,12 +92,19 @@ IPCache::IPCache(const std::string& path)
 
 void IPCache::SetPath(const std::string& path) {
   Thread::LockGuard guard(path_mutex_);
-  path_ = path;
+  if (path != path) {
```

Comment:
is it path_ != path ?

Reply:
good call!

Reply:
Failed test was at: https://github.com/cilium/proxy/actions/runs/14383419488/job/40334342351?pr=1274

Reply:
ah right, I didn't see any failure till now, let me take a look

Reply:
Maybe these tests were never run with flow validation on? Could be simple as that?

Reply:
> Maybe these tests were never run with flow validation on? Could be simple as that?

True. The upstream cilium/cilium didn't have flow validation enabled for any workflow. We can disable it for the time being here as well.